core/src/com/unciv/ui/trade/DiplomacyScreen.kt

```diff
@@ -114,6 +115,15 @@ class DiplomacyScreen(val viewingCiv:CivilizationInfo):CameraStageBaseScreen() {
             diplomacyTable.add(allyString.toLabel()).row()
         }
 
+        val protectors = otherCiv.getProtectorCivs()
+        if (protectors.size > 0) {
+            var protectorString = "{Protected by}: ".tr()
+            for (protectorName in protectors.map{it.civName}) {
+                protectorString = protectorString + "{$protectorName} ".tr()
```

Comment:
This would look like: "Protected by: America Germany Japan"
Maybe add a "," so it looks like "Protected by: America, Germany, Japan"
Something like 
```
var protectorString = "{Protected by}: "+ protectors.map { it.civName }.joinToString(", ")
```
You don't even need to .tr() anything since the .toLabel() below does that :)

Reply:
Done. Thanks for the suggestion O_-